hw13/trigrams.py

```diff
@@ -0,0 +1,36 @@
+import random
+
+
+d = {}
+
+
+def read_text(lines):
+    for x in range(0, len(lines) - 1):
+        words = lines[x].lower().split()
+        for y in range(0, len(words) - 2):
+            d[(words[y], words[y + 1])] = [words[y + 2]]
```

Comment:
This is replacing the last value in the dictionary for each key pair. You need to be appending the list each time the word pair appears in the text.